src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Snackbar/SnackBar.android.cs

```diff
@@ -27,11 +28,21 @@ internal async ValueTask Show(VisualElement sender, SnackBarOptions arguments)
 				snackBar.SetAnchorView(renderer.View);
 			}
 
+			var shape = snackBar.View.Background as GradientDrawable;
 			if (arguments.BackgroundColor != Forms.Color.Default)
 			{
-				snackBarView.SetBackgroundColor(arguments.BackgroundColor.ToAndroid());
+				shape?.SetColor(arguments.BackgroundColor.ToAndroid().ToArgb());
 			}
 
+			shape?.SetCornerRadii(new float[]
```

Comment:
Wouldn't this break default SnackBar style on Android?

https://material.io/components/snackbars#specs

Reply:
It wouldn't break if the default corner radius is set to 4.
```xml
 <shape android:shape="rectangle"
    xmlns:android="http://schemas.android.com/apk/res/android">
    <solid android:color="#323232"/>
    <corners android:radius="4dp"/>
</shape>
```

I've reverted the changes and set it to 4 by default

Reply:
Compared behavior before and after this PR: 

![image](https://user-images.githubusercontent.com/6609929/123998573-46de3e00-d9da-11eb-93c9-5c6c328b74a6.png)

Everything seems fine, but corner radius too small (it's actually set to 4, I checked). Default value feels more like `10` to me.
Here's code from this PR, but default is 10:

![image](https://user-images.githubusercontent.com/6609929/123999673-72155d00-d9db-11eb-9b66-80fcc17763ac.png)

It's possible that default value is dependent on pixel density or something.

Reply:
Set it to 10, I don't think it is a big issue, because customers can update it to any value. The main idea that it is now consistent between all platforms.

Reply:
Thank you for setting default to 10. Yes, but to set it to 10 otherwise would require passing an `options` argument to every snackbar, which is no very convenient. 

Could you also not apply corner radius at all if it's a default one (same thing you do with color)?
Some customers may already have their configuration in `xml`, or maybe this default wouldn't work correctly in some cases. In any way, this would be less of a breaking change if it wouldn't be applied unless explicitly requested:

![image](https://user-images.githubusercontent.com/6609929/124012159-82343900-d9e9-11eb-95e1-e59511601c5a.png)

Reply:
I kind of agree. I think we should not set a default at all tbh. We should preserve the corner radius as it was before this change, but just add the ability for the user to now change it. That way, if a user upgrades there is no change unless they explicitly start using this option. Else, there will be a visual change, users will be frustrated and they have to implement code to set it back to how they wanted it to be.

Reply:
@jfversluis The thing is, Android has good default value (4dp), but all other platforms didn't have one at all (since they don't have native SnackBar control). And the issue was that SnackBar is looking bad on iOS.

Maybe we can make default exactly match Android's one, and then apply it to all platforms for consistency.
If you think, this would be too much of a breaking change, this can be released with next major version.

Reply:
That is the problem with library development. Once it's in, every change in default behavior is a breaking change. In this case it's a rather small one, but still. People now might have custom code in place to overcome this, if we suddenly start changing default values that might have unexpected behavior that people will get just by upgrading a library and they will have a hard time, spending valuable time, figuring out why that is.

Again, this is a pretty small one so I can let this one slide I guess, but then let's at least preserve the default behavior on Android. Although I don't see why we can't just set no default besides the fact that it will look nicer out of the box.

Reply:
To satisfy all opinions: Set Corner radius to the Android default value (4dp). Calculate density and convert the value to pixel so the radius displayed correctly.

Reply:
But that's only for Android now, right?

Reply:
Other platforms didn't have the "default" corner radius. So to be consistent between all platforms (Android as etalon) all other platforms will have a corner radius by default. To be honest, who doesn't like the rounded corners. Even Windows 11 has it now))

Reply:
We're just gonna roll with it, let's do it!